### PR TITLE
12.0 mig account payment partner

### PR DESCRIPTION
**Pull request by mourad-ehm**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Can you please CIs status and squash commit history a bit (at least Transbot commits)

File: account_payment_partner/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+# © 2014 Akretion - Alexis de Lattre <alexis.delattre@akretion.com>
+# © 2014 Tecnativa - Pedro M. Baeza
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+{
+    'name': 'Account Payment Partner',
+    'version': '1.0.0',
```
Comment (sergio-teruel) [inline account_payment_partner/__manifest__.py]:
```suggestion
    'version': '12.0.1.0.0',
```

File: account_payment_partner/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
```
Comment (sergio-teruel) [inline account_payment_partner/__manifest__.py]: Remove all utf-8 in py files

**Review by sergio-teruel — CHANGES_REQUESTED** [review]

**Comment by js-landoo** [comment]
Hi @mourad-ehm 
You have this PR nearly finished. Can you have a look?
@sergio-teruel has indicated needed small changes

It's the same question with  #509 

Many thanks

**Comment by pedrobaeza** [comment]
I have:

* squashed commit history as it wasn't done at 11.0.
* pre-generated README
* fixed test for this version
* remove old migration scripts
* integrated latest v11 changes

so I proceed to merge this one as well and continue with the rest.

**Pull request merged**